specification/vmware/Microsoft.AVS/models.tsp

```diff
@@ -901,16 +895,11 @@ model ClusterUpdate {
 
 @doc("The properties of a cluster that may be updated")
 model ClusterUpdateProperties {
-  ...DisplayNameProperty;
-
   @doc("The cluster size")
   clusterSize?: int32;
 
   @doc("The hosts")
   hosts?: string[];
-
-  @doc("Name of the vsan datastore associated with the cluster")
-  vsanDatastoreName?: string;
```

Comment:
The changes were add `DisplayNameProperty` to a private cloud and cluster properties as well as adding `vsanDatastoreName?` to cluster properties.